aws/resource_aws_iam_server_certificate.go

```diff
@@ -209,6 +212,23 @@ func resourceAwsIAMServerCertificateImport(
 	return []*schema.ResourceData{d}, nil
 }
 
+func currentlyInUseBy(awsErr string, meta interface{}) {
+	r := regexp.MustCompile(`currently in use by ([a-z0-9:-]+)\/([a-z0-9-]+)\.`)
+	matches := r.FindStringSubmatch(awsErr)
+	if len(matches) > 0 {
+		lbName := matches[2]
+		conn := meta.(*AWSClient).elbconn
+		describeElbOpts := &elb.DescribeLoadBalancersInput{
+			LoadBalancerNames: []*string{aws.String(lbName)},
+		}
+		if _, err := conn.DescribeLoadBalancers(describeElbOpts); err != nil {
+			if elbErr, ok := err.(awserr.Error); ok && elbErr.Code() == "LoadBalancerNotFound" {
```

Comment:
Nitpick: This error check here can be simplified to

```go
if isAWSErr(err, "LoadBalancerNotFound", "") {
```
We unfortunately haven't changed it everywhere yet, since introducing this helper function - but that's the long-term goal.